blog/service/impl/src/test/kotlin/com/excref/kotblog/blog/service/category/CategoryServiceIntegrationTest.kt

```diff
@@ -39,6 +40,18 @@ class CategoryServiceIntegrationTest : AbstractServiceIntegrationTest() {
     }
 
     @Test
+    fun getByUuids() {
+        // given
+        val category = helper.persistCategory()
+        val category2 = helper.persistCategory()
+        val categories = listOf<Category>(category, category2)
```

Comment:
I believe you don't need to specify type argument of generic param, because of type inference